src/features/import/components/Upload/UploadFile.tsx

```diff
@@ -0,0 +1,177 @@
+import { UploadFileOutlined } from '@mui/icons-material';
+import { useDropzone } from 'react-dropzone';
+import {
+  Box,
+  Button,
+  CircularProgress,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import messageIds from '../../l10n/messageIds';
+import theme from 'theme';
```

Comment:
Instead of importing `theme`, use `useTheme()` to get it from the context. That way, if the theme changes, it will just work.

This won't really make a difference today, but if we want to introduce something like dark mode it will make all the difference.